api-sdk/src/main/java/com/smartling/api/sdk/file/parameters/FileUploadParameterBuilder.java

```diff
@@ -110,6 +111,18 @@ public FileUploadParameterBuilder fileType(final FileType fileType)
         return this;
     }
 
+    /**
+     * library UID that uses sdk for sending files
+     * if UID is not provided default UID that is equal to sdk library UID is set
+     * @param clientUid
+     * @return
+     */
+    public FileUploadParameterBuilder clientUid(final String clientUid)
```

Comment:
We discussed it with Dima. Dima told that it should be String. If some connector send some this in some other format we should not care about it.
Though we can discuss it one more time.